pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "setuptools_scm", "wheel>=0.36.1", "Cython>=0.29.28"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scalene"
+description = "Scalene: A high-resolution, low-overhead CPU, GPU, and memory profiler for Python"
+license = {text = "Apache License 2.0"}
+keywords = ["performance memory profiler"]
+# version is computed by setup.py
+dynamic = ["version"]
+requires-python = ">= 3.8"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: IPython",
+    "Framework :: Jupyter",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Debuggers",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+]
+readme = "README.md"
+authors = [
+    {name = "Emery Berger", email = "emery@cs.umass.edu"},
+    {name = "Sam Stern"},
+    {name = "Juan Altmayer Pizzorno"},
```

Comment:
Another nice feature of `pyproject.toml` is that it makes it easy to have multiple authors. I've taken the liberty of adding the other authors listed in the project's README :sweat_smile: